### DISCUSSION TITLE
盐酸的C++基础知识小讲堂——lambda表达式 | Berksey

### DISCUSSION THREAD

**Discussion by NachtgeistW** [NachtgeistW/Berksey#38] [General]
https://nachtgeistw.github.io/Berksey/c++/2019/12/22/lambda/ 

 真实基础知识，真实讲堂 以下笔记摘自 D.N.Code 群聊天记录

**Comment by Chlorie** [comment]
有道理，我去灌一篇blog吧（问题发言

**Comment by NachtgeistW** [comment]
> @Chlorie
> 有道理，我去灌一篇blog吧（问题发言

灌，都可以灌，写多点写多点

**Comment by Dyaosy** [comment]
关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（

**Comment by Chlorie** [comment]
> @Dyaosy
> 关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（

这样的吗，像这种“C++xx 新加的特性”我都 take for granted 了，经常说“你为什么不用 xxx”结果一看人家是低版本语言（

**Comment by NachtgeistW** [comment]
> 关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（

我的 blog 居然真的有人看 /facepalm

**Comment by Dyaosy** [comment]
> @NachtgeistW
> > 关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（
> 
> 我的 blog 居然真的有人看 /facepalm

夜轮姐姐人又有才，说话又好听，我超喜欢的。
（夜轮...啊啊啊...跑圈！！！）

**Comment by Chlorie** [comment]
> @NachtgeistW
> > 关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（
> 
> 我的 blog 居然真的有人看 /facepalm

我的就真的没人看 [快哭了][快哭了][快哭了][快哭了][快哭了]

**Comment by Dyaosy** [comment]
> @Chlorie
> > @NachtgeistW
> > > 关于最后vs编译器能识别lambda返回类型的问题，这是c++14指定的新特性，所以两本primer都没写（
> > 
> > 我的 blog 居然真的有人看 /facepalm
> 
> 我的就真的没人看 [快哭了][快哭了][快哭了][快哭了][快哭了]

我看过了（（（
盐酸太🐮了！我没啥好说的！
顺便夜轮blog就是在盐酸blog里翻到的（X
